<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulatorWidget</class>
 <widget class="QWidget" name="SimulatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Define simulation parameters:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;- white noise: &lt;/span&gt;standard deviation of gaussian distributed noise in [picoseconds] &lt;/p&gt;&lt;p&gt;&lt;img src=&quot;:/icons/icons/warning.png&quot; width=&quot;30&quot; height=&quot;30&quot;/&gt; This parameter is defined per station and differs from a baseline based noise by a factor of sqrt(2)! --&amp;gt; the default value corresponds to a white noise of 25ps per observation.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;- clock ASD:&lt;/span&gt; Allan standard deviation of clock accuracy in [picoseconds]&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;- clock dur:&lt;/span&gt; equivalent time of clock ASD in [minutes]&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;- tropo Cn:&lt;/span&gt; tropospheric turbolence parameter in [10^-7 m^-1/3] &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;- tropo H:&lt;/span&gt; troposhperic height in [meters]&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;- tropo dh:&lt;/span&gt; tropospheric height increment in [meters]&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;- tropo dhseg:&lt;/span&gt; tropospheric time increment in [hours]&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;- tropo zwd0:&lt;/span&gt; additional constant zenith wet delay in [millimeters]&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;- tropo ve:&lt;/span&gt; wind velocity towards eastern direction in [meters/second]&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;- tropo vn:&lt;/span&gt; wind velocity towards northern direction in [meters/second]&lt;/p&gt;&lt;p&gt;use a fixed seed to reproduce your simulation result. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>simulations:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_simulations">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cn">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Import &lt;span style=&quot; font-weight:700;&quot;&gt;Cn&lt;/span&gt; and &lt;span style=&quot; font-weight:700;&quot;&gt;v&lt;/span&gt; values from:&lt;br/&gt;&lt;span style=&quot; font-family:'Merriweather Sans','Helvetica Neue','Helvetica','Arial','sans-serif'; color:#222222; background-color:#ffffff;&quot;&gt;Schartner, M. &lt;/span&gt;&lt;span style=&quot; font-family:'Merriweather Sans','Helvetica Neue','Helvetica','Arial','sans-serif'; font-weight:700; color:#222222; background-color:#ffffff;&quot;&gt;Deriving a global troposphere model for space geodetic simulations based on an ML ensemble featuring uncertainty quantification.&lt;/span&gt;&lt;span style=&quot; font-family:'Merriweather Sans','Helvetica Neue','Helvetica','Arial','sans-serif'; color:#222222;&quot;/&gt;&lt;span style=&quot; font-family:'Merriweather Sans','Helvetica Neue','Helvetica','Arial','sans-serif'; font-style:italic; color:#222222;&quot;&gt;J Geod&lt;/span&gt;&lt;span style=&quot; font-family:'Merriweather Sans','Helvetica Neue','Helvetica','Arial','sans-serif'; color:#222222;&quot;/&gt;&lt;span style=&quot; font-family:'Merriweather Sans','Helvetica Neue','Helvetica','Arial','sans-serif'; font-weight:700; color:#222222;&quot;&gt;99&lt;/span&gt;&lt;span style=&quot; font-family:'Merriweather Sans','Helvetica Neue','Helvetica','Arial','sans-serif'; color:#222222;&quot;&gt;, 72 (2025). &lt;/span&gt;&lt;a href=&quot;https://doi.org/10.1007/s00190-025-01996-w&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://doi.org/10.1007/s00190-025-01996-w&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Appendix A and B (updated for new stations).&lt;/p&gt;&lt;p&gt;Full list of values can be found at &lt;a href=&quot;https://github.com/TUW-VieVS/VieSchedpp_AUTO/tree/master/SIMULATION_CATALOGS&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://github.com/TUW-VieVS/VieSchedpp_AUTO/tree/master/SIMULATION_CATALOGS &lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;&lt;br/&gt;If a station is not listed, use 1.8 for Cn and 8 for v. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;You can derive your own values for new locations based on the data found in &lt;a href=&quot; https://gpc.ethz.ch/products/Troposphere/Cn/multi-year%20monthly/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://gpc.ethz.ch/products/Troposphere/Cn/multi-year%20monthly/.&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Cn and v from Schartner (2025)</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_zwd">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Import &lt;span style=&quot; font-weight:700;&quot;&gt;zwd&lt;/span&gt; based on the model from:&lt;br/&gt;&lt;span style=&quot; font-family:'-apple-system','BlinkMacSystemFont','Segoe UI','Roboto','Oxygen-Sans','Ubuntu','Cantarell','Helvetica Neue','sans-serif'; color:#333333; background-color:#fcfcfc;&quot;&gt;Crocetti, L., Schartner, M., Wareyka-Glaner, M.F.&lt;/span&gt;&lt;span style=&quot; font-family:'-apple-system','BlinkMacSystemFont','Segoe UI','Roboto','Oxygen-Sans','Ubuntu','Cantarell','Helvetica Neue','sans-serif'; font-style:italic; color:#333333;&quot;&gt;et al.&lt;/span&gt;&lt;span style=&quot; font-family:'-apple-system','BlinkMacSystemFont','Segoe UI','Roboto','Oxygen-Sans','Ubuntu','Cantarell','Helvetica Neue','sans-serif'; color:#333333;&quot;&gt; ZWDX: a global zenith wet delay forecasting model using XGBoost. &lt;/span&gt;&lt;span style=&quot; font-family:'-apple-system','BlinkMacSystemFont','Segoe UI','Roboto','Oxygen-Sans','Ubuntu','Cantarell','Helvetica Neue','sans-serif'; font-style:italic; color:#333333;&quot;&gt;Earth Planets Space&lt;/span&gt;&lt;span style=&quot; font-family:'-apple-system','BlinkMacSystemFont','Segoe UI','Roboto','Oxygen-Sans','Ubuntu','Cantarell','Helvetica Neue','sans-serif'; font-weight:700; color:#333333;&quot;&gt;76&lt;/span&gt;&lt;span style=&quot; font-family:'-apple-system','BlinkMacSystemFont','Segoe UI','Roboto','Oxygen-Sans','Ubuntu','Cantarell','Helvetica Neue','sans-serif'; color:#333333;&quot;&gt;, 163 (2024). &lt;/span&gt;&lt;a href=&quot;https://doi.org/10.1186/s40623-024-02104-6&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://doi.org/10.1186/s40623-024-02104-6&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;&lt;br/&gt;If a station is not listed, use 150 for zwd0. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>zwd from Crocetti (2024)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_seed">
       <property name="text">
        <string>fixed seed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_seed">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-2147483647</number>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_wnTable">
       <property name="text">
        <string>*show wn table</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget_simpara">
     <property name="toolTip">
      <string>define simulation parameters</string>
     </property>
     <property name="statusTip">
      <string>define simulation parameters</string>
     </property>
     <column>
      <property name="text">
       <string>Station</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>white noise*</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>clock ASD</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>clock dur</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>tropo Cn</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>tropo H</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>tropo dh</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>tropo dhseg</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>tropo ve</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>tropo vn</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>tropo zwd0</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_obs_minus_comp">
       <property name="text">
        <string>output observed minus computed</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_seed</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_seed</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>834</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
